compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -65,8 +65,11 @@ error.attempt.expose.non.public.symbol=\
 error.invokable.must.return=\
   this {0} must return a result
   
-error.main.cannot.be.public=\
-  the main function cannot be explicitly marked as public
+error.main.cannot.be.non.public=\
+  the main function cannot be non public
```

Comment:
This message feels bit weird. I think `main function should be declared as a public function`, `main function should be public` like message would be more suitable. Because it also have the details on how to fix the issue. WDYT?

Reply:
thought of "the main function should be public", but went with this since it didn't look consistent. Will change.

Reply:
@MaryamZi better to change the key as well, to avoid the double-negative